libraries/botbuilder-lg/src/generated/LGFileLexer.ts

```diff
@@ -8,266 +8,266 @@
  * Licensed under the MIT License.
  */
 
-import { ATN } from "antlr4ts/atn/ATN";
-import { ATNDeserializer } from "antlr4ts/atn/ATNDeserializer";
-import { CharStream } from "antlr4ts/CharStream";
-import { Lexer } from "antlr4ts/Lexer";
-import { LexerATNSimulator } from "antlr4ts/atn/LexerATNSimulator";
-import { NotNull } from "antlr4ts/Decorators";
-import { Override } from "antlr4ts/Decorators";
-import { RuleContext } from "antlr4ts/RuleContext";
-import { Vocabulary } from "antlr4ts/Vocabulary";
-import { VocabularyImpl } from "antlr4ts/VocabularyImpl";
+import { ATN } from 'antlr4ts/atn/ATN';
+import { ATNDeserializer } from 'antlr4ts/atn/ATNDeserializer';
+import { CharStream } from 'antlr4ts/CharStream';
+import { Lexer } from 'antlr4ts/Lexer';
+import { LexerATNSimulator } from 'antlr4ts/atn/LexerATNSimulator';
+import { NotNull } from 'antlr4ts/Decorators';
+import { Override } from 'antlr4ts/Decorators';
+import { RuleContext } from 'antlr4ts/RuleContext';
+import { Vocabulary } from 'antlr4ts/Vocabulary';
+import { VocabularyImpl } from 'antlr4ts/VocabularyImpl';
 
-import * as Utils from "antlr4ts/misc/Utils";
+import * as Utils from 'antlr4ts/misc/Utils';
 
 
 export class LGFileLexer extends Lexer {
-	public static readonly NEWLINE = 1;
-	public static readonly OPTION = 2;
-	public static readonly COMMENT = 3;
-	public static readonly IMPORT = 4;
-	public static readonly TEMPLATE_NAME_LINE = 5;
-	public static readonly MULTILINE_PREFIX = 6;
-	public static readonly TEMPLATE_BODY = 7;
-	public static readonly INVALID_LINE = 8;
-	public static readonly MULTILINE_SUFFIX = 9;
-	public static readonly ESCAPE_CHARACTER = 10;
-	public static readonly MULTILINE_TEXT = 11;
-	public static readonly MULTILINE_MODE = 1;
+    public static readonly NEWLINE = 1;
+    public static readonly OPTION = 2;
+    public static readonly COMMENT = 3;
+    public static readonly IMPORT = 4;
+    public static readonly TEMPLATE_NAME_LINE = 5;
+    public static readonly MULTILINE_PREFIX = 6;
+    public static readonly TEMPLATE_BODY = 7;
+    public static readonly INVALID_LINE = 8;
+    public static readonly MULTILINE_SUFFIX = 9;
+    public static readonly ESCAPE_CHARACTER = 10;
+    public static readonly MULTILINE_TEXT = 11;
+    public static readonly MULTILINE_MODE = 1;
 
-	// tslint:disable:no-trailing-whitespace
-	public static readonly channelNames: string[] = [
-		"DEFAULT_TOKEN_CHANNEL", "HIDDEN",
-	];
+    // tslint:disable:no-trailing-whitespace
+    public static readonly channelNames: string[] = [
+        'DEFAULT_TOKEN_CHANNEL', 'HIDDEN',
+    ];
 
-	// tslint:disable:no-trailing-whitespace
-	public static readonly modeNames: string[] = [
-		"DEFAULT_MODE", "MULTILINE_MODE",
-	];
+    // tslint:disable:no-trailing-whitespace

```

Comment:
should these warning disable still be there?

Reply:
It's not part of the script, I think we should leave it that way.

Reply:
Probably, but we could check if the warning is necessary after all the fixes